content/05.use-cases-c.md

```diff
@@ -35,13 +35,13 @@ Additionally, if the research software is kept on only the researcher's hard dri
 When the project owner is finished with the project, or their contract expires, there generally should be a handover period of this software in order for the next cohort of researchers to reuse what was already developed [@doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7].
 GitHub facilitates project continuity among research software and research code by providing tools that make this handover period easier. 
 As we have already mentioned, using Git for code in Ecology and Evolution can allow for a "paper trail" of sorts to be created for the research software, thus allowing for future owners of the code access to the entire history of the project [@doi:10.1038/s41559-017-0160]. 
-Additionally, GitHub allows for repositories and organizations to have designated Code Owners [@https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners]; these code owners can change through time allowing for the transition of research software from one cohort of researchers to the next [@doi:10.1890/ES14-00402.1]. There are also multiple means by which to archive package version information with GitHub. Most simply, package versions can be included in README files. More sophisticated recording options include adding an enviornment.yml and/or an environment.txt file, or using tools such as Packrat [@https://rstudio.github.io/packrat/] to store all packages in the repository itself. This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
+Additionally, GitHub allows for repositories and organizations to have designated Code Owners [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners]; these code owners can change through time allowing for the transition of research software from one cohort of researchers to the next [@doi:10.1890/ES14-00402.1]. There are also multiple means by which to archive package version information with GitHub. Most simply, package versions can be included in README files. More sophisticated recording options include adding an environment.yml and/or an environment.txt file, or using tools such as Packrat (https://rstudio.github.io/packrat/) to store all packages in the repository itself. This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
```

Comment:
Packrat has been superseded by [`renv`](https://rstudio.github.io/renv/articles/renv.html).  Citation for `renv`: 
```
@Manual{,
    title = {renv: Project Environments},
    author = {Kevin Ushey},
    year = {2022},
    note = {R package version 0.15.4},
    url = {https://CRAN.R-project.org/package=renv},
  }
```

Also, I think Packrat was mentioned somewhere else, because I already suggested this change in another PR.  This might be redundant or maybe just conflicts with an open PR?

Reply:
Hmmm, not sure if it's mentioned elsewhere.  Should I just remove the mention of packrat all together?